storage-manager/src/PrefixCache.cpp

```diff
@@ -593,13 +595,17 @@ int PrefixCache::ifExistsThenDelete(const string& key)
     else  // let makeSpace() delete it if it's already in progress
       return 0;
   }
-  bool journalExists = bf::exists(journalPath);
+  auto kvStorage = KVStorageInitializer::getStorageInstance();
+  auto tnx = kvStorage->createTransaction();
+  auto resultPair = tnx->get(journalName);
+
+  bool journalExists = resultPair.first;
   // assert(objectExists == bf::exists(cachedPath));
 
-  size_t objectSize = (objectExists ? bf::file_size(cachedPath) : 0);
+  //size_t objectSize = (objectExists ? bf::file_size(cachedPath) : 0);
```

Comment:
remove if you don't need this.

Reply:
Sure, I will do in in patch related to `journal to meta`